test/integration/kubernetes_deploy_test.rb

```diff
@@ -1080,6 +1081,24 @@ def test_not_apply_resource_can_be_pruned
     ])
   end
 
+  def test_network_policies_are_deployed_first
+    assert_deploy_success(deploy_fixtures("network-policy"))
```

Comment:
Instead of creating a separate fixture set with just this in it, on this line you can do `deploy_fixtures('hello-cloud', subset: ['network_policy.yml'])`